gestalt-module/src/main/java/org/terasology/gestalt/module/ModuleServiceRegistry.java

```diff
@@ -6,9 +6,7 @@
 
 public class ModuleServiceRegistry extends ServiceRegistry {
     public ModuleServiceRegistry(PermissionProviderFactory permissionProviderFactory) {
-        with(ModulePathScanner.class);
         with(ModuleMetadataJsonAdapter.class);
-        with(ModulePathScanner.class);
```

Comment:
I removed `ModulePathScanner` from the default context because on Android you might want to use `AndroidModulePathScanner` instead. This would not be possible if the concrete base class is always provided.